src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/MailPostAction.java

```diff
@@ -0,0 +1,14 @@
+package pl.tomaszdziurko.jvm_bloggers.mailing;
+
+/**
+ * This interface is intended to use in tests only and probably should be moved from here. <br/>
+ *
+ * @author Adam Dec
+ * @see pl.tomaszdziurko.jvm_bloggers.mailing.LogMailSender
+ * @since 0.7.0
```

Comment:
`@since` should ba avoided according to [general coding guidelines](https://github.com/tdziurko/jvm-bloggers/blob/2dc3dc51e2c2af50d1a8829777c050e9e6190ec5/CONTIBUTING.md#general-coding-guidelines).